cmd/argo/commands/cron/create.go

```diff
@@ -81,6 +81,14 @@ func CreateCronWorkflows(filePaths []string, cliOpts *cliCreateOpts, submitOpts
 			log.Fatal(err)
 		}
 		cronWf.Spec.WorkflowSpec = newWf.Spec
```

Comment:
This seems hacky,  it your comment suggests there is a good reason. Maybe add a code comment here?

Reply:
Thanks. Added comment.